lib/utils/packet.c

```diff
@@ -0,0 +1,398 @@
+#include "packet.h"
+#include "rte_mbuf_core.h"
+
+#include <netinet/in.h>
+#include <rte_ether.h>
+#include <rte_ip.h>
+#include <rte_mbuf.h>
+#include <rte_tcp.h>
+#include <rte_udp.h>
+
+#include <assert.h>
+#include <stdlib.h>
+
+#include <rte_build_config.h>
+#include <string.h>
+
+#include "lib/dataplane/packet/packet.h"
+
+////////////////////////////////////////////////////////////////////////////////
+
+static struct rte_mbuf *
+make_mbuf4(
```

Comment:
Can we merge these functions to avoid seeding havoc?
https://github.com/yanet-platform/yanet2/blob/482815b9d543b3ea8c6b8e092fafdccbffec88b2/lib/dataplane/module/testing.c#L6